metadata-ingestion/tests/integration/s3/test_s3.py

```diff
@@ -6,6 +6,15 @@
 
 import moto.s3
 import pytest
+
+# Check if profiling dependencies are available
+try:
+    import pydeequ  # noqa: F401
+    import pyspark  # noqa: F401
+
+    _PROFILING_ENABLED = True
+except ImportError:
+    _PROFILING_ENABLED = False
```

Comment:
seems like the same code is present in another files, can we extract the logic to some common class?

Reply:
Please see another comment, I think we should get rid of this approach at all.

Reply:
removed too.